administrator/components/com_banners/src/Model/BannerModel.php

```diff
@@ -389,7 +393,7 @@ protected function preprocessForm(Form $form, $data, $group = 'content')
 	 */
 	public function save($data)
 	{
-		$input = Factory::getApplication()->input;
+		$app = Factory::getApplication();
```

Comment:
This change is not needed, because you are using its input only. This is the case in several places, please revert all of them.